src/index.html

```diff
@@ -17,6 +21,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo-container">
+        <a href="index.html">
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+            class="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav__container">
+        <ul>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link is-active"
+              href="index.html"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
Add empty lines between multiline sibling blocks of HTML for better readability. There should be an empty line between each `<li>` block.